config/checkstyle/suppressions.xml

```diff
@@ -5,11 +5,5 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
```

Comment:
You can safely remove this file if there aren't any more suppressions that you need.

Reply:
Thank for letting me know. I afraid checkstyle stop running if we remove suppressions file because a property in checkstyle rules using that file. Will remove it in the next version.

Reply:
I check again and it works fine without suppression file